deltacat/compute/compactor/model/delta_annotated.py

```diff
@@ -286,8 +286,14 @@ def _split_single(delta_annotated: DeltaAnnotated) -> List[DeltaAnnotated]:
 
                         result.append(new_da)
                 else:
+                    logger.info(
+                        f"Split was not performed on correct delta with locator: {delta_annotated.locator}"
```

Comment:
What does `correct delta` refer to here?

Reply:
I wanted this to be different log message than the below one. Changing this to more meaningful.